### PR TITLE
Swati iiit/python code

### PR DESCRIPTION
**Pull request by swatiIIIT**
### **Describe your change:**

* [x ] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### **Checklist:**
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [x] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: strings/insert_char_after_every_k_characters.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+def insert_character_after_every_k_elements(input_string, k, character_to_insert):
```
Comment (s39674) [inline strings/insert_char_after_every_k_characters.py:1]:
```suggestion
def insert_character_after_every_k_elements(input_string: str, PlaceChrEvery: int, character_to_insert: str) -> str:
```

File: strings/insert_char_after_every_k_characters.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
+def insert_character_after_every_k_elements(input_string, k, character_to_insert):
+    """
+    Add charater after every k characters in given string
+    >>> insertCharacterAfterEveryKelements("abcdef",2,'*')
+    ['*abcdef', 'ab*cdef', 'abcd*ef']
+    >>> insertCharacterAfterEveryKelements("abcdef",2,'*')
+    ['*abcdef', 'abcd*ef']
+    """
+    result = []
+    for i in range(0, len(input_string), k):
```
Comment (s39674) [inline strings/insert_char_after_every_k_characters.py:10]:
```suggestion
    for i in range(0, len(input_string), PlaceChrEvery):
```

File: strings/insert_char_after_every_k_characters.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    ['*abcdef', 'abcd*ef']
+    """
+    result = []
+    for i in range(0, len(input_string), k):
+        result.append(input_string[:i] + character_to_insert + input_string[i:])
+    return str(result)
+
+
+input_string = input('Enter the string ')
+k = int(input('Enter after which K element want to enter the string '))
```
Comment (s39674) [inline strings/insert_char_after_every_k_characters.py:16]:
```suggestion
PlaceChrEvery = int(input('Enter after which element want to enter the string '))
```

File: strings/insert_char_after_every_k_characters.py [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+"""
+    k defines after which character we want to insert the given character in the string
+"""
+character_to_insert = input('Enter the character to insert in the string ')
+
+result = insert_character_after_every_k_elements(input_string, k, character_to_insert)
```
Comment (s39674) [inline strings/insert_char_after_every_k_characters.py:22]:
```suggestion
result = insert_character_after_every_k_elements(input_string, PlaceChrEvery, character_to_insert)
```

**Review by s39674 — CHANGES_REQUESTED** [review]
I think that should take care of it.

**Comment by cclauss** [comment]
In preparation for that denial-of-service attack that is [Hacktoberfest](https://hacktoberfest.com), I am closing all pull requests that do not have passing tests.

**Pull request closed without merging**